src/test/resources/config/bs.json

```diff
@@ -0,0 +1,184 @@
+{
```

Comment:
Let us have either json or yaml configuration.

Reply:
removed the json file